### DISCUSSION TITLE
GPS api

### DISCUSSION THREAD

**Discussion by jhpgfs** [zepp-health/zeppos-forum#54] [Q&A] [answered]
Hi all,

I would like to develop an APP for sailing navigation.
Just a basic one for now: Show Lat, Long, SOG and COG. 

// This is not enough for me
import { Distance } from '@zos/sensor'

Is there any other way to obtain more gps data ?

Thanks,

José

**Comment by sirAvent** [comment]
There will be a GPS API for Zepp OS 2.1. Documentation is still being worked on.

**Comment by jhpgfs** [comment]
Hi,

Thanks for answering me.
Do you have a time frame for Zepp OS 2.1. release ?
Can I offer my free collaboration in any way ?

Kind regards,

José

**Reply by dlaraf** [reply]
Hi @sirAvent  ! Any update? Also interested

**Comment by MyoungXUE** [comment, marked as answer]
Sorry for the long wait, but we don't have a geolocation API yet.

Please be patient, we plan to release the geolocation API in Zepp OS 2.1.

Zepp OS 2.0 devices will have the opportunity to upgrade to version 2.1, please wait for the announcement of the upgrade plan.

**Comment by dlaraf** [comment]
Ok, thank you for the quick replay. Looking forward to see it

**Comment by joaquinperaza** [comment]
I'm also interested on having an open location API GPS+altimeter

**Comment by T-Bone82** [comment]
same here for developping a parking location reminder app
